containers/provision_ip_discovery_agent/start.go

```diff
@@ -49,7 +49,12 @@ func init() {
 }
 
 func runStartCmd(cmd *cobra.Command, args []string) {
-	flag.Set("logtostderr", "true")
+	var err error
+	err = flag.Set("logtostderr", "true")
```

Comment:
fixed to pass new  ci checks. failed since flag.Set returns error with now check for it